### DISCUSSION TITLE
Installing on Mac

### DISCUSSION THREAD

**Discussion by DavidJinho** [nielsmaerten/ynab-buddy#38] [Q&A] [answered]
Hi Niels,

I'm trying to install YNAB Buddy on my mac however when I open the downloaded file it just opens in text editor.

Can you tell me what I'm doing wrong?

Thanks!
David

**Comment by nielsmaerten** [comment, marked as answer]
Hey David,

I think this is because the MacOS doesn't know yet that the file is a runnable program.  ynab-buddy needs to be run form the Terminal.

This might help: https://support.apple.com/guide/terminal/make-a-file-executable-apdd100908f-06b3-4e63-8a87-32e71241bab4/mac
